client/src/document/ingredients/browser-compatibility-table/utils.ts

```diff
@@ -191,3 +193,20 @@ export function getCurrentSupport(
   // Default (likely never reached)
   return getFirst(support);
 }
+
+export function getPreviousVersion(
+  version: BCD.VersionValue | string | undefined,
+  browser: BCD.BrowserStatement
+): BCD.VersionValue | string | undefined {
+  if (browser && typeof version === "string") {
+    const browserVersions = Object.keys(browser["releases"]).sort(
+      compareVersions
+    );
+    const currentVersionIndex = browserVersions.indexOf(version);
+    if (currentVersionIndex && currentVersionIndex > 0) {
```

Comment:
`Array.indexOf` should always return a `number`:

```suggestion
    if (currentVersionIndex > 0) {
```